src/main/java/de/retest/recheck/ui/diff/ElementDifference.java

```diff
@@ -29,7 +29,7 @@
 
 	protected final LeafDifference identifyingAttributesDifference;
 
-	protected Collection<ElementDifference> childDifferences = new ArrayList<>();
+	protected List<ElementDifference> childDifferences = new ArrayList<>();
```

Comment:
Why do you use a list? Do we need the index access? If not a set should suffice. If you use a set (which may also have a iterating order, as `LInkedHashSet` and `TreeSet` ensures), I would suggest to use a `TreeSet`, which does the sorting for you.

Reply:
Since `Element` is sorted, I'd like to point that out here. This is much more directly done by using a list.

Reply:
Ok